src/Formatter.php

```diff
@@ -0,0 +1,75 @@
+<?php
+declare(strict_types = 1);
+
+namespace LanguageServer;
+
+use LanguageServer\Protocol\{TextEdit, Range, Position, ErrorCode};
+use AdvancedJsonRpc\ResponseError;
+
+class Formatter
+{
+
+    /**
+     *
+     * @param string $content            
+     * @param string $uri            
+     *
+     * @return \LanguageServer\Protocol\TextEdit[]
+     * @throws \AdvancedJsonRpc\ResponseError
+     */
+    public function format(string $content, string $uri)
+    {
+        $path = uriToPath($uri);
+        $cs = new \PHP_CodeSniffer();
+        $cs->initStandard($this->findConfiguration($path));
```

Comment:
> Will this also work in environments where
> 
> URIs are not file URIs

For directory URI it wont work if URI = directory with config file but in this case it will load default configuration. Should we be worried about such cases? Do you notices URIs different than file?

> there is no config file in the project ?

If config file is not found PSR2 standard is loaded.